Apps/NotSensorLogger2/gradle.properties

```diff
@@ -0,0 +1,22 @@
+# Project-wide Gradle settings.
+# IDE (e.g. Android Studio) users:
+# Gradle settings configured through the IDE *will override*
+# any settings specified in this file.
+# For more details on how to configure your build environment visit
+# http://www.gradle.org/docs/current/userguide/build_environment.html
+# Specifies the JVM arguments used for the daemon process.
+# The setting is particularly useful for tweaking memory settings.
+org.gradle.jvmargs=-Xmx2048m -Dfile.encoding=UTF-8
+# When configured, Gradle will run in incubating parallel mode.
+# This option should only be used with decoupled projects. More details, visit
+# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
+# org.gradle.parallel=true
+# AndroidX package structure to make it clearer which packages are bundled with the
+# Android operating system, and which are packaged with your app's APK
+# https://developer.android.com/topic/libraries/support-library/androidx-rn
+android.useAndroidX=true
+# Enables namespacing of each library's R class so that its R class includes only the
+# resources declared in the library itself and none from the library's dependencies,
+# thereby reducing the size of the R class for that library
+android.nonTransitiveRClass=true
+MAPBOX_DOWNLOADS_TOKEN=sk.eyJ1IjoieWFzaG5hbGx1cmkiLCJhIjoiY2x0M3JwdTl5MjBzczJybXVpMjAyZzh6MSJ9.XBRqrPDGkbq4-H33eYHZug
```

Comment:
the secret key is supposed to be setup in your pc's gradle property. Remove it from this file and mention about[ this setup](https://docs.mapbox.com/android/maps/guides/install/#configure-your-secret-token) in readme